toolkit/resources/dracutmodules/90mountesppartition/mountesppartition-generator.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+set -x
+set -e
+
+echo "Running mountesppartition-generator.sh" > /dev/kmsg
+
+# type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+function updateVeritySetupUnit () {
+    systemdDropInDir=/etc/systemd/system
+    verityDropInDir=$systemdDropInDir/systemd-veritysetup@root.service.d
+
+    mkdir -p $verityDropInDir
+    verityConfiguration=$verityDropInDir/verity-azl-extension.conf
+
+    cat <<EOF > $verityConfiguration
+[Unit]
+After=espmountmonitor.service
```

Comment:
You can use `Before` and `RequiredBy` / `WantedBy` instead of `After` and `Requires` to specify the same dependencies but from the opposite service/mount file.